assets/airports/eddt.json

```diff
@@ -313,7 +313,7 @@
           "method": "random",
           "rate": 7,
           "airlines": [
-              ["n/fastga", 5],
+              ["n/fastGA", 5],
```

Comment:
id be more inclined to use `fastga` here instead of `fastGA`.  but for the purposes of the hotfix, this should do it.